source/frontend/src/features/mapSideBar/acquisition/tabs/subFiles/table/SubFilesResultsTable.tsx

```diff
@@ -0,0 +1,28 @@
+import { Table } from '@/components/Table';
+import { ApiGen_Concepts_AcquisitionFile } from '@/models/api/generated/ApiGen_Concepts_AcquisitionFile';
+
+import { createSubFilesTableColumns } from './colums';
+
+export interface ISubFilesResultsTableProps {
+  results: ApiGen_Concepts_AcquisitionFile[];
+  currentAcquisitionFileId: number;
+  routeURL: string;
+}
+
+export function SubFilesResultsTable(props: ISubFilesResultsTableProps) {
+  const { results, ...rest } = props;
+  const columns = createSubFilesTableColumns(props.currentAcquisitionFileId, props.routeURL);
+
+  return (
+    <Table<ApiGen_Concepts_AcquisitionFile>
+      name="AcquisitionCompensationTable"
```

Comment:
nit: name should be adjusted to match sub-files (not compensations)

Reply:
Updated